boa/src/builtins/object/mod.rs

```diff
@@ -406,6 +408,28 @@ pub fn make_object(_: &mut Value, args: &[Value], ctx: &mut Interpreter) -> Resu
     Ok(object)
 }
 
+/// Uses the SameValue algorithm to check equality of objects
+pub fn is(_: &mut Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
+    let x = args.get(0).expect("Cannot get object").clone();
+    let y = args.get(1).expect("Cannot get object").clone();
```

Comment:
```suggestion
    let x = args.get(0).cloned().unwap_or_else(Value::undefined);
    let y = args.get(1).cloned().unwap_or_else(Value::undefined);
```
Here if a value is not specified it should be `undefined`, we should not panic if one or two of the parameters was not sepecified.